app/assets/stylesheets/components/_checkbox-filter.scss

```diff
@@ -0,0 +1,150 @@
+.filter-form,
```

Comment:
This class is also used in existing finder-frontend UI: https://github.com/alphagov/finder-frontend/search?utf8=%E2%9C%93&q=filter-form&type=

These styles may inadvertently style things in other finders.

You could scope it to `main.search` to avoid this risk and we'll refactor later.

Reply:
Extracted to separate CSS file so it is not included on non-site search pages